testsuite/integration-arquillian/tests/other/console/src/main/java/org/keycloak/testsuite/console/page/users/UserAttributesForm.java

```diff
@@ -121,8 +126,40 @@ public void setValues(UserRepresentation user) {
         setLastName(user.getLastName());
         if (user.isEnabled() != null) setEnabled(user.isEnabled());
         if (user.isEmailVerified() != null) setEmailVerified(user.isEmailVerified());
+        if (user.getGroups() != null && user.getGroups().length > 0) setGroups(new HashSet<String>(user.getGroups()));
         if (user.getRequiredActions() != null) setRequiredActions(new HashSet<>(user.getRequiredActions()));
     }
 
     // TODO Contact Information section
+
+    public class GroupSelect extends MultipleStringSelect2 {
```

Comment:
I'm getting some compilation errors here. The tests need to be passing with Java 8 (which is used by our CI).

Reply:
fixed in latest commit